src/hotspot/share/gc/shared/verifyOption.hpp

```diff
@@ -26,11 +26,15 @@
 #define SHARE_GC_SHARED_VERIFYOPTION_HPP
 
 enum VerifyOption {
-      VerifyOption_Default = 0,
+  VerifyOption_Default = 0,
 
-      // G1
-      VerifyOption_G1UsePrevMarking = VerifyOption_Default,
-      VerifyOption_G1UseFullMarking = VerifyOption_G1UsePrevMarking + 1
+  // G1
+
+  // Use "prev" mark bitmap information using pTAMS.
+  VerifyOption_G1UsePrevMarking = VerifyOption_Default,
+  // Use "next" mark bitmap information from full gc marking. This does not
+  // use (or need) TAMS.
+  VerifyOption_G1UseFullMarking = VerifyOption_G1UsePrevMarking + 1
```

Comment:
I suggest that as a followup this should be an enum class, with the `VerifyOption_` prefix dropped from the enumerators, instead using `VerifyOption::G1UsePrevMarking` (for example).